packages/component-library/components/NavigationBar/NavigationBar.tsx

```diff
@@ -69,8 +69,8 @@ class NavigationBar extends React.Component<Props, unknown> {
 
     return (
       <nav className={styles.links}>
-        {Object.keys(children).map(key =>
-          this.renderNavSection(key, children[key])
+        {(Object.keys(children) as Array<keyof typeof children>).map(key =>
+          this.renderNavSection(key, children[key] || [])
```

Comment:
This is a logical and not purely type base change because there is an actual issue. All children props could be undefined, but the renderNavSection expects it to be defined.